articles/ros2_security_contexts.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: default
+title: ROS 2 Security Contexts
+permalink: articles/ros2_security_contexts.html
+abstract:
+  This article specifies the integration between security and contexts.
+author:  >
+  [Ruffin White](https://github.com/ruffsl),
+  [Mikael Arguedas](https://github.com/mikaelarguedas)
+
+published: false
+categories: Security
+---
+
+{:toc}
+
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+TODO: Some concise overview introduction here
+
+## Concepts
+
+Before detailing the SROS 2 integration of the contexts, the following concepts are introduced.
+
+### Participant
+
+Participant is the object representing a single entity on the network, in the case of DDS the ``Participant`` is a DDS DomainParticipant and the object to which a set of access control and security identity apply.
```

Comment:
This could be phrased a little clearer.

```suggestion
Participant is the object representing a single entity on the network. In the case of DDS, the ``Participant`` is a DDS DomainParticipant, which has both access control permissions and a security identity.
```